@here/olp-sdk-dataservice-read/lib/StatisticsClient.ts

```diff
@@ -0,0 +1,31 @@
+import { DataStoreContext } from "./DataStoreContext";
+
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+/**
+ * A class that provides possibility to get Statistic Metadata and Data for Versioned layer
+ */
+export class StatisticsClient {
+    public readonly context: DataStoreContext;
+
+    constructor(context: DataStoreContext) {
```

Comment:
Can we use like this: constructor(private readonly context: DataStoreContext) {  } ?

Reply:
fixed